src/options.cpp

```diff
@@ -65,7 +65,7 @@ namespace po = boost::program_options;
 "Assign this device to a device group."
 
 #define PRODUCTION_HELP \
-"Enable a production release."
+"Allow pushed updates (production updates) on the device."
```

Comment:
I think this new help message takes it somewhat sidelines.

The PRODUCTION flag does not enable pushed updates.
What it does is: designate any devices as production devices when they are registered using a version of `lmp-dev-reg` built with this flag. That way these devices get a completely different subset of updates from regular CI (test) devices. These are not "pushed updates" - they are simply "production updates".

IIUC this option is then enabled on a per-branch level in the factory using this switch:
https://github.com/foundriesio/meta-lmp/blob/main/meta-lmp-base/recipes-sota/lmp-device-register/lmp-device-register_git.bb#L15

I think @doanac or @mike-sul can help more here to understant how this plays out with the factory's CI configs.

Reply:
Nevertheless, I'm not sure I'd like to mention "production updates" explicitly here.

Right now the only difference between the test vs production device is indeed the set of updates they get.
But, that difference can be extended in the future.
Also, our users are free to extend that difference right now, as they can read and react to this attribute.

Therefore, I'd prefer to keep it more flexible like this: `Designate|Mark the device as a production device.`

Reply:
I don't think anyone fixes it on a per-branch setting. They leave it as a run-time option for devices. I guess the help message should be something like: "flag device to receive updates via waves"

Reply:
I understood the main difference to be that the device didnt need to ping periodically for its updates but that the update to take would be pushed instead. is this not the case? how shall I reword the option then?

Reply:
I think one of these options are OK:
- `Mark the device as a production device.` - this is my preference.
- `Enable production updates on the device.` - this gives a more exact context, but already too many details IMO.

Reply:
IIRC we operate a term "production device" extensively in our docs/blogs by now.
So, it should be clear enough to a user what it means.